src/commands/api/delete.js

```diff
@@ -0,0 +1,63 @@
+const { flags } = require('@oclif/command')
+const { deleteApi } = require('../../requests/api')
+const { getApiIdentifierArg, splitPathParams } = require('../../support/command/parse-input')
+const BaseCommand = require('../../support/command/base-command')
+const inquirer = require('inquirer')
+
+class DeleteAPICommand extends BaseCommand {
+
+  async run() {
+    const { args, flags } = this.parse(DeleteAPICommand)
+    const apiPath = getApiIdentifierArg(args)
+    const [owner, name, version] = splitPathParams(apiPath)
+
+    if (version) {
+      this.logCommandSuccess = this.setSuccessMessage('deletedApiVersion')
+    } else if (!flags.force && await this.confirmDeletion(name) == false) {
```

Comment:
We're testing explicitly for `false` here, which means undefined/null/etc will still pass as okay. Best practice is to be more inclusive and test for falsey-ness.
A possible edge case: What if the user hits Cmd/Ctrl-C during this prompt, will the command execute or bail out?

Reply:
I just tested this, it bails out. I think its because the default response for `confirmDeletion` is `false`. So anything other than responding with `y` is `false`.

Reply:
@ponelat Would `!= true` or `!== true` be a better option? Do you have any recommendation for this type of the method return comparision?

Reply:
`!== true` would be better (given the await, wrapping in in brackets is kinda meh... ie: `&& !(await this.confirmDeletion(name)`)
The thinking is that if-and-only-if it's confirmed (ie: true), go and delete. If anything else, don't.
Thanks for testing @conor-ward .

### Side note on `==` vs `===`
In short, always use `===` (and `!==`).
The end :joy: 

- `==` will try coerce to same type, so `1 == '1' // true`
- `===` has two quirks, if you don't want them you'll need Object.is. `NaN === NaN // false. wtf` but `Object.is(NaN, NaN) // true`